lg_common/src/lg_common/uscs_service.py

```diff
@@ -61,37 +61,38 @@ def __init__(self,
 
         self.state = self._grab_state(initial_state_scene_url)
 
-        if not self.on_offline_state:
+        if self.state is None:
+            self.state = USCSMessageResponse()
+
+        if self.on_offline_state is None:
             """
             By default, when offline, go to initial state if there is no offline state
             explicitly specifyed.
             """
             self.on_offline_state = self.state
```

Comment:
that's OK - we should handle offline state at all time. I still cant imagine a case where we dont want to do that.